### DISCUSSION TITLE
payload 2.x maintenance plan

### DISCUSSION THREAD

**Discussion by finkinfridom** [payloadcms/payload#9332] [General]
Payload 3.0 has just been released.
We tried to understand the changes on our projects and they're impacting lot of files (for some of them, probably we should also completely rewrite from scratch).
As a question to everybody: are there any plan to maintain the 2.x releases going (for bug-fix, improvements, ...)?

Thanks

**Comment by immotus** [comment]
@finkinfridom, as far as I can tell, there are no plans to add new features or even provide long-term maintenance to v2. The Payload team has recently cleared a whole lot of issues, marking that new stuff won't be ported back to v2.

I have completed the migration to v3 on one of my pet projects, but with Next.js, it is now unsatisfactorily slow.

**Reply by AlessioGr** [reply]
> but with Next.js, it is now unsatisfactorily slow.

We have some big performance improvements coming to 3.2.0 - stay tuned!

**Reply by immotus** [reply]
@AlessioGr looking forward to it!

**Comment by DanRibbens** [comment]
We haven't set an end of life date for Payload 2.x. We plan to keep up with any critical updates and security vulnerabilities going forward.
